src/main/java/org/broadinstitute/consent/http/service/InstitutionService.java

```diff
@@ -17,27 +17,42 @@ public InstitutionService(InstitutionDAO institutionDAO) {
   }
 
   public Integer createInstitution(String name, String itDirectorName, String itDirectorEmail, Integer createUser) {
-    return this.institutionDAO.insertInstitution(name, itDirectorName, itDirectorEmail, createUser, new Date());
+    Date date = new Date();
+    return institutionDAO.insertInstitution(name, itDirectorName, itDirectorEmail, createUser, date);
+  }
+
+  public Integer createInstitution(String name, String itDirectorName, String itDirectorEmail, Integer createUser, Date date) {
```

Comment:
Address errors about incorrect number of matchers in service class by versions of the create and update methods that include all parameters including date